airflow/exceptions.py

```diff
@@ -106,10 +106,21 @@ def __init__(self, value: Any) -> None:
         super().__init__(value)
         self.value = value
 
+
+class UnmappableXComTypePushed(UnmappableXComPushed):
+    """Raise when an unmappable type is pushed."""
+
     def __str__(self) -> str:
         return f"unmappable return type {type(self.value).__qualname__!r}"
 
 
+class UnmappableXComSizePushed(UnmappableXComPushed):
+    """Raise when the pushed value is to large to map."""
+
+    def __str__(self) -> str:
+        return f"unmappable return value size: {len(self.value)}"
```

Comment:
It might be worth including the limit value in the message too.

Reply:
Done